src/kitaspect.cpp

```diff
@@ -72,10 +72,19 @@ void KitAspect::setAsn1Exe(Kit *k, const Utils::FilePath &v)
 QVariant KitAspect::defaultValue(const Kit *k) const
 {
     Q_UNUSED(k);
-    const QString path = Core::ICore::libexecPath() + QLatin1String("/asn1scc/asn1.exe");
+    const QString path = Core::ICore::libexecPath().toString() + QLatin1String("/asn1scc/asn1scc");
```

Comment:
why? binary in the package is made by us, and we usually kept the asn1.exe name

Reply:
because in the latest asn1scc release asn1.exe changed name to asn1scc
reverted